llmproxy/config/internal.config.yml

```diff
@@ -86,10 +90,11 @@ available_models:
 
   - name: Vertexai
     class: llmproxy.provider.google.vertexai.VertexAI
+    max-output-tokens: 50
     models:
       - name: text-bison
-        cost_per_token_input: 0.0000015
-        cost_per_token_output: 0.000002
+        cost_per_token_input: 0.0000005
+        cost_per_token_output: 0.0000005
```

Comment:
Missing Category information; This is fine you can do 2 different PRs, one for category and one for cost, but the task on the Kanban board encompasses both